cosmos/job/drm/__init__.py

```diff
@@ -1 +1,10 @@
 __author__ = 'egafni'
+
+import importlib
+import os
+import pkgutil
+
+# Imports all classes within DRM so that the __subclasses__ attribute is populated
```

Comment:
more commentary on why __subclass__ is needed please

Reply:
Actually on further thought, I don't think this `__subclass__` stuff is worth the complexity.   It's over-engineered to me and I think makes the code less readable than the original dict.  I know it saves adding an element to the dict when adding a new DRM, but this is a very rare event

Reply:
See line 23 in cosmos/job/drm/DRM_Base.py. It's necessary to validate the DRM options dict.

Reply:
Ah okay, let's keep it as is then, just a bit more on the comments then